aim/web/ui_v2/src/pages/Metrics/MetricsContainer.tsx

```diff
@@ -1,9 +1,50 @@
 import React from 'react';
-
 import Metrics from './Metrics';
 
 function MetricsContainer(): React.FunctionComponentElement<React.ReactNode> {
-  return <Metrics />;
+  const tableRef = React.useRef<HTMLDivElement>(null);
+  const chartRef = React.useRef<HTMLDivElement>(null);
+  const wrapperRef = React.useRef<HTMLDivElement>(null);
+
+  React.useEffect(() => {
+    return () => {
+      document.removeEventListener('mousemove', startResize);
+      document.removeEventListener('mouseup', endResize);
+    };
+  }, []);
+
+  const handleResize = React.useCallback(() => {
+    document.addEventListener('mousemove', startResize);
+    document.addEventListener('mouseup', endResize);
+  }, []);
+
+  const startResize = React.useCallback((event: MouseEvent): void => {
+    const containerHeight: number =
+      (tableRef.current as HTMLDivElement)?.getBoundingClientRect()?.height +
```

Comment:
@arsengit is using the `as` keyword necessary here?
Cause RefObject type is already mentioned above via generic.